pkg/apis/settings/settings.go

```diff
@@ -0,0 +1,89 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package settings
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/configmap"
+
+	"github.com/aws/karpenter-core/pkg/operator/settingsstore"
```

Comment:
The dependency arrow from apis -> operator seems like it may cause circular issues in the future. Is there a way to draw the arrow the other way?

Reply:
Good comment, updated.